packages/block-library/src/button/style.scss

```diff
@@ -1,7 +1,7 @@
 $blocks-button__height: 56px;
 
 .wp-block-button {
-	color: $white;
+	color: var(--wp--color--white, $white);
```

Comment:
This covers the case where `--wp--color--white` is not present, so it fallbacks to `$white`. However, we also need to cover cases where the browser doesn't have support for CSS Custom Properties and write something like:

```css
.wp-block-button {
	color: $white;
	color: var(--wp--color--white, $white);
}
```

which can be done automatically with some [plugins](https://postcss.github.io/postcss-custom-properties/).

Reply:
We also don't want to do this until we're certain this is the way to go.